packages/core/src/app/db/range-bucket.ts

```diff
@@ -122,13 +122,13 @@ export class BaseRangeBucket implements RangeBucket {
    * @param end The end of the range.
    * @returns true if start > end, false otherwise.
    */
-  private validateRange(start: BigNum, end: BigNum): void {
+  private validateRange(start: BigNumber, end: BigNumber): void {
     // Make sure start is less than end
     if (!start.lt(end)) {
       throw new Error('Start not less than end')
     }
     // Make sure start is greater than or equal to zero
-    if (!start.gten(0)) {
+    if (!start.gte(ZERO)) {
```

Comment:
The main reason that `ZERO` and `ONE` exist, other than convenience, is to avoid allocating memory over and over again for very common numbers. My thinking is that there shouldn't be any naming conflicts with other types of numbers because all other numbers that we would use should be primitives, so there's no benefit in creating constants for them. Happy to rename if you think I should, though 😄

Reply:
> all other numbers that we would use should be primitives

Ahh good point :)